doc/source/user_guide/tutorials/mesh/explore_mesh.rst

```diff
@@ -0,0 +1,283 @@
+            from ansys.dpf.core import examples
+            # Import the operators module
+            from ansys.dpf.core import operators as ops
+
+            # Define the result file path
+            result_file_path_1 = examples.find_static_rst()
+            # Create the model
+            model_1 = dpf.Model(data_sources=result_file_path_1)
+            # Get the mesh
+            meshed_region_1 = model_1.metadata.meshed_region
+
+    .. tab-item:: LSDYNA
+
+        .. jupyter-execute::
+
+            # Import the ``ansys.dpf.core`` module
+            from ansys.dpf import core as dpf
+            # Import the examples module
+            from ansys.dpf.core import examples
+            # Import the operators module
+            from ansys.dpf.core import operators as ops
+
+            # Define the result file path
+            result_file_path_2 = examples.download_d3plot_beam()
+            # Create the DataSources object
+            ds_2 = dpf.DataSources()
+            ds_2.set_result_file_path(filepath=result_file_path_2[0], key="d3plot")
+            ds_2.add_file_path(filepath=result_file_path_2[3], key="actunits")
+            # Create the model
+            model_2 = dpf.Model(data_sources=ds_2)
+            # Get the mesh
+            meshed_region_2 = model_2.metadata.meshed_region
+
+    .. tab-item:: Fluent
+
+        .. jupyter-execute::
+
+            # Import the ``ansys.dpf.core`` module
+            from ansys.dpf import core as dpf
+            # Import the examples module
+            from ansys.dpf.core import examples
+            # Import the operators module
+            from ansys.dpf.core import operators as ops
+
+            # Define the result file path
+            result_file_path_3 = examples.download_fluent_axial_comp()["flprj"]
+            # Create the model
+            model_3 = dpf.Model(data_sources=result_file_path_3)
+            # Get the mesh
+            meshed_region_3 = model_3.metadata.meshed_region
+
+    .. tab-item:: CFX
+
+        .. jupyter-execute::
+
+            # Import the ``ansys.dpf.core`` module
+            from ansys.dpf import core as dpf
+            # Import the examples module
+            from ansys.dpf.core import examples
+            # Import the operators module
+            from ansys.dpf.core import operators as ops
+
+            # Define the result file path
+            result_file_path_4 = examples.download_cfx_mixing_elbow()
+            # Create the model
+            model_4 = dpf.Model(data_sources=result_file_path_4)
+            # Get the mesh
+            meshed_region_4 = model_4.metadata.meshed_region
+
+Explore the mesh data
+---------------------
+
+You can access the mesh data by manipulating the |MeshedRegion| object methods.
+The mesh data includes :
+
+- Unit;
+- Nodes, elements and faces;
+- Named selections: .
+
+When instantiating nodes, elements, faces and named selections you get the corresponding DPF objects:
```

Comment:
I don't think instantiating is the right word. Maybe more "calling the `nodes`, `elements [...] properties" ?